R/plotD3_aggregated_profiles_explainer.R

```diff
@@ -151,7 +154,7 @@ plotD3.aggregated_profiles_explainer <- function(x, ..., size = 2, alpha = 1,
                   yMax = yMax + yMargin, yMin = yMin - yMargin,
                   yMean = y_mean, labelNames = label_names,
                   size = size, alpha = alpha, color = color,
-                  onlyNumerical = only_numerical,
+                  onlyNumerical = ifelse(variable_type == "numerical", TRUE, FALSE),
```

Comment:
you can use `variable_type == "numerical"` instead of `ifelse(variable_type == "numerical", TRUE, FALSE)`